consensus/istanbul/backend/backend.go

```diff
@@ -316,3 +352,87 @@ func (sb *backend) HasBadProposal(hash common.Hash) bool {
 func (sb *backend) Close() error {
 	return nil
 }
+
+// IsQIBFTConsensus returns whether qbft consensus should be used
+func (sb *backend) IsQIBFTConsensus() bool {
+	// If qibftBlock is not defined in genesis, then use legacy ibft
+	if sb.config.QibftBlock == nil {
+		return false
+	}
+
+	if sb.qibftConsensusEnabled || sb.config.QibftBlock.Uint64() == 0 {
+		return true
+	}
+
+	if sb.chain != nil && sb.chain.CurrentHeader().Number.Cmp(sb.config.QibftBlock) >= 0 {
+		return true
+	}
+	return false
+}
+
+// IsQIBFTConsensusCrossed returns whether qbft consensus block has crossed
+// This is useful to see when the snapshot headers should be applied
+func (sb *backend) IsQIBFTConsensusCrossed() bool {
```

Comment:
this code is a duplicate of `IsQIBFTConsensus`, except of `>= 1` instead of `>= 0`. can we not re-use the same function with an argument to avoid duplication of most of the code?

Reply:
Done, pushed a commit

Reply:
Actually removed `IsQIBFTConsensusCrossed()` as it looks it might not be needed